### ISSUE TITLE
The link is broken

### ISSUE THREAD

**Issue by ramsayleung** [marioortizmanero/rspotify-bench#1]
The conclusion at the bottom is broken

> Read more details at nullderef.com/blog/rspotify-upgrade.

The link is 404

**Comment by marioortizmanero** [comment]
Yes I haven't uploaded it yet (or even finished writing it). I've just released this a bit early so that the creator of https://github.com/ramsayleung/rspotify/issues/235 could try the patched version of rspotify here: https://github.com/marioortizmanero/rspotify-bench/tree/master/rspotify-0.10.0-patched.

Thanks for the report, I'll let you know when it's ready :)
